helpers/class.TestSession.php

```diff
@@ -530,14 +530,16 @@ public function moveBack($allowTimeout = false)
     /**
      * Skip the current item.
      *
+     * @param bool $allowUntouchedItems whether to allow skipping items in state INITIAL (not ever seen yet by the TT)
```

Comment:
I'm not I get what it means. to skip items in initial mode... Can you clarify the comment?

Reply:
By default, items can be submitted only after they are initialized (started). For auto-skipping, we need to also allow submitting those items not ever started

Reply:
Got it, thanks! :)